apps/schema/src/components/ProfileManager.vue

```diff
@@ -0,0 +1,343 @@
+<template>
+  <div class="container-fluid">
+    <MessageError v-if="error">error: {{ error }} {{ search }}</MessageError>
+    <div class="row">
+      <div class="col-4">
+        <div class="sticky-top" style="max-height: 100vh; overflow: auto">
+          <p>
+            available profiles:
+            <span
+              class="badge mr-1"
+              :style="{ 'background-color': color(profile) }"
+              v-for="profile in profiles.filter(
+                (profile) => !selectedProfiles.includes(profile)
+              )"
+              @click="selectedProfiles.push(profile)"
+              >{{ profile }}</span
+            >
+          </p>
+          <p v-if="selectedProfiles?.length > 0">
+            selected profiles:
+            <span
+              class="badge mr-1"
+              :style="{ 'background-color': color(profile) }"
+              v-for="profile in selectedProfiles"
+              @click="
+                selectedProfiles.splice(selectedProfiles.indexOf(profile), 1)
+              "
+              >{{ profile }}</span
+            >
+          </p>
+          <InputSearch v-model="search" />
+          <ul class="list-group sticky-top">
+            <li
+              v-for="table in tables"
+              class="list-group-item list-group-item-action"
+              :class="{
+                active:
+                  table.selected || table.columns?.some((c) => c.selected),
+              }"
+              v-scroll-to="{
+                el: '#' + (table.name ? table.name.replaceAll(' ', '_') : ''),
+                offset: -50,
+              }"
+            >
+              <h5>
+                {{ table.name }}
+                <sup v-if="table.profiles"
+                  ><span
```

Comment:
Try to get the closing > on the same line as the sup tag. Once you do it by hand the autoformatter should not change it back to this ;)